app/javascript/packages/document-capture/components/in-person-location-step.tsx

```diff
@@ -29,64 +32,20 @@ interface FormattedLocation {
   sundayHours: string;
   weekdayHours: string;
 }
-
-interface RequestOptions {
-  /**
-   * Whether to send the request as a JSON request. Defaults to true.
-   */
-  json?: boolean;
-
-  /**
-   * Whether to include CSRF token in the request. Defaults to true.
-   */
-  csrf?: boolean;
-
-  /**
-   * Optional. HTTP verb used. Defaults to GET.
-   */
-  method?: string;
+interface LocationQuery {
```

Comment:
since the locationQuery and location interfaces have the same fields can we re-use one interface by pulling into a separate file?